tests/Controller/ExportControllerTest.php

```diff
@@ -4,917 +4,192 @@
-                'count' => 20,
-                'exceptionMessage' => sprintf(
-                    'Required argument "shopkey" was not given, or does not match the shopkey schema "%s"',
-                    ''
-                )
-            ],
-            'Malformed shopkey provided' => [
-                'shopkey' => 'ABCD01815',
-                'start' => 1,
-                'count' => 20,
-                'exceptionMessage' => sprintf(
-                    'Required argument "shopkey" was not given, or does not match the shopkey schema "%s"',
-                    'ABCD01815'
-                )
-            ],
-            '"count" parameter is zero' => [
-                'shopkey' => $this->validShopkey,
-                'start' => 1,
-                'count' => 0,
-                'exceptionMessage' => 'The value 0 is not greater than zero'
-            ],
-            '"count" parameter is some string' => [
-                'shopkey' => $this->validShopkey,
-                'start' => 'some string',
-                'count' => 20,
-                'exceptionMessage' => 'The value "some string" is not a valid numeric'
-            ],
-            '"count" parameter is a negative number' => [
-                'shopkey' => $this->validShopkey,
-                'start' => 1,
-                'count' => -1,
-                'exceptionMessage' => 'The value -1 is not greater than zero'
-            ],
-            '"start" parameter is some string' => [
-                'shopkey' => $this->validShopkey,
-                'start' => 'some string',
-                'count' => 20,
-                'exceptionMessage' => 'The value "some string" is not a valid numeric'
-            ],
-            '"start" parameter is a negative number' => [
-                'shopkey' => $this->validShopkey,
-                'start' => -1,
-                'count' => 20,
-                'exceptionMessage' => 'The value -1 is not greater than or equal to zero'
-            ],
-        ];
-    }
+        $product = $this->createVisibleTestProduct();
+        $this->enableFindologicInPluginConfiguration();
 
-    /**
-     * @dataProvider invalidArgumentProvider
-     * @throws InconsistentCriteriaIdsException
-     * @throws UnknownShopkeyException
-     */
-    public function testExportWithInvalidArguments(
-        string $shopkey,
-        $start,
-        $count,
-        string $exceptionMessage
-    ): void {
-        $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessage($exceptionMessage);
-
-        /** @var SalesChannelContext|MockObject $salesChannelContextMock */
-        $salesChannelContextMock = $this->getDefaultSalesChannelContextMock();
-
-        /** @var Request $request */
-        $request = new Request(['shopkey' => $shopkey, 'start' => $start, 'count' => $count]);
-
-        $this->exportController->export($request, $salesChannelContextMock);
+        $response = $this->sendRequest();
```

Comment:
It's necessary to check the code which request gets send, maybe rename to `callExportUrl`?

Reply:
This request has nothing to do with the export at all. It is a general purpose class, that allows to send requests to the application.

Reply:
If it's a general purpose I would expect to at least pass the url to the method.

Reply:
Sorry that was actually my bad. I will rename this.

Reply:
Renamed it to `sendExportRequest`.